projects/ADuCM3029_no_os_blinky_example/src/blinky_test.c

```diff
@@ -0,0 +1,71 @@
+/*****************************************************************************
+ * ADuCM3029_no_os_test_project.c
+ *****************************************************************************/
+
+#include <sys/platform.h>
```

Comment:
it seems that blinky_test and blinky_example are the same thing. Remove one of them.